ivy_tests/test_ivy/test_frontends/test_onnx/test_elementwise.py

```diff
@@ -0,0 +1,227 @@
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         input=x[0],
+#     )
+
+# @handle_frontend_test(
+#     fn_tree="onnx.acos",
+#     dtype_and_x=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("float"),
+#     ),
+# )
+# def test_onnx_acos(
+#     *,
+#     dtype_and_x,
+#     on_device,
+#     fn_tree,
+#     frontend,
+#     test_flags,
+# ):
+#     input_dtype, x = dtype_and_x
+#     helpers.test_frontend_function(
+#         input_dtypes=input_dtype,
+#         frontend=frontend,
+#         test_flags=test_flags,
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         input=x[0],
+#     )
+#
+# @handle_frontend_test(
+#     fn_tree="onnx.acosh",
+#     dtype_and_x=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("float"),
+#     ),
+# )
+# def test_onnx_acosh(
+#     *,
+#     dtype_and_x,
+#     on_device,
+#     fn_tree,
+#     frontend,
+#     test_flags,
+# ):
+#     input_dtype, x = dtype_and_x
+#     helpers.test_frontend_function(
+#         input_dtypes=input_dtype,
+#         frontend=frontend,
+#         test_flags=test_flags,
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         input=x[0],
+#     )
+#
+# @handle_frontend_test(
+#     fn_tree="onnx.add",
+#     dtype_and_x=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("numeric"),
+#         num_arrays=2,
+#         large_abs_safety_factor=2.5,
+#         small_abs_safety_factor=2.5,
+#         safety_factor_scale="log",
+#     ),
+#     alpha=st.integers(min_value=1, max_value=5),
+# )
+# def test_onnx_add(
+#     *,
+#     dtype_and_x,
+#     alpha,
+#     on_device,
+#     fn_tree,
+#     frontend,
+#     test_flags,
+# ):
+#     input_dtype, x = dtype_and_x
+#     helpers.test_frontend_function(
+#         input_dtypes=input_dtype,
+#         frontend=frontend,
+#         test_flags=test_flags,
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         rtol=1e-03,
+#         input=x[0],
+#         other=x[1],
+#         alpha=alpha,
+#     )
+#
+# @handle_frontend_test(
+#     fn_tree="onnx.asin",
+#     dtype_and_x=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("float"),
+#     ),
+# )
+# def test_onnx_asin(
+#     *,
+#     dtype_and_x,
+#     on_device,
+#     fn_tree,
+#     frontend,
+#     test_flags,
+# ):
+#     input_dtype, x = dtype_and_x
+#     helpers.test_frontend_function(
+#         input_dtypes=input_dtype,
+#         frontend=frontend,
+#         test_flags=test_flags,
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         input=x[0],
+#     )
+
+
+def test_onnx_abs_v2():
+    x = ivy.random_uniform(low=-100.0, high=0, shape=10)
+
+    onnx_abs = onnx.abs(x)
+    torch_abs = torch.abs(x)
+    jax_abs = jax.lax.abs(x)
+    paddle_abs = paddle.abs(x)
+    tf_abs = tf.abs(x)
+
+    all_list = [onnx_abs, jax_abs, paddle_abs, tf_abs]
+
+    assert torch.allclose(torch_abs, all_list)
+
```

Comment:
Also, instead of using `torch.allclose` I would suggest using [`value_test`](https://github.com/unifyai/ivy/blob/c1c24051fdc5856ad14e222ca20a9f30dc653b1b/ivy_tests/test_ivy/helpers/assertions.py#L81). This checks not only for values, but for shapes and dtypes as well.

Reply:
Hi @AnnaTz

So i have tried using the `helpers.value_test`, but the issue with it is that you can only compare one framework with another framework. So that means I will only be able to compare the onnx operator result with another framework like torch and not all the frameworks.

The next commit I will make shows that using the value_test for the onnx.abs method actually works, but it can only be tested with one framework.

Reply:
We have to check all backends one after another anyway to test for all cases, so calling `helpers.value_test` should work.

Reply:
So this new commit makes use of the `@given` and the `helpers.value_test` method, but the helper function takes in 2 arrays and also a ground truth backend.

The first array is obviously ONNX, while the second is PyTorch and the ground truth backend I used was also PyTorch

Reply:
When testing a frontend function we should be comparing the return of the function to that of the original function. 
So here we want to import our onnx frontend and the original onnx and compare between them. There is no reason to use torch at all.
Also the `ground_truth_backend` argument is only used to print correct error messages. In the frontend testing case we always pass the name of the frontend, in this case "onnx".